bus_sdk/publishers/event_publisher.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+
+
+class EventPublisher(ABC):
+    @property
+    @abstractmethod
+    def name(self) -> str:
+        raise NotImplementedError
+
+    @abstractmethod
+    def publish(self, message: dict) -> str:
```

Comment:
is it `dict` for a reason or should also be `Message`?

Reply:
Was left like that initially to be more flexible, plus `Message` inherits from `dict` so syntactically it was OK. Though with the design choice of per type messages e.g. `ProfileMessage` I have now changed the abstract arg to `Message`.